pastis/algorithms.py

```diff
@@ -9,6 +16,88 @@
 
 max_iter = 5
 
+###############################################################################
```

Comment:
I think this should be moved to another module, probably the io.py module.
Also check out the way I've documented the functions of this module: it uses numpydoc, which allows to build nice html documentation. If you don't know numpydoc, I can also fix the documentation later..